src/main/java/ch/wisv/events/WebConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package ch.wisv.events;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebConfiguration implements WebMvcConfigurer {
+
+    @Override
+    public void configurePathMatch(PathMatchConfigurer configurer) {
+      configurer.setUseTrailingSlashMatch(true);
```

Comment:
Seems like a good temporary fix. 

As described in https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#web-application-changes, we should manually define all paths with and without trailing slashes, or change the Servlet/Web filter. I am unable to find examples of the latter.

Lets stick to this temporary solution, but make a new issue describing this.

Reply:
Created a new issue for this.